### ISSUE TITLE
Installing without UI

### ISSUE THREAD

**Issue by zarif98** [spyder-ide/spyder#22260]
The Python installer has a feature where you can run the installer without even having to see the GUI. Is it possible for Spyder to do the same thing? 

Here's the Python example: https://docs.python.org/3.9/using/windows.html#installing-without-ui

**Comment by ccordoba12** [comment]
Hey @zarif98, thanks for reporting. Do you want to run our Windows or Mac installer like that?

**Comment by zarif98** [comment]
@ccordoba12 Just the Windows installer for now. We have no use for MacOS at the moment.

**Comment by ccordoba12** [comment]
Ok, thanks for the extra info.

@dalthviz, could you explain @zarif98 how to run our installer in batch mode? Thanks!

**Comment by dalthviz** [comment]
Hi @zarif98 to install on Windows without seeing a GUI/in silent mode you need to use the `/S` flag. For more info on the installer command line arguments you can check https://github.com/spyder-ide/spyder/blob/5.x/installers/Windows/README.md

**Comment by zarif98** [comment]
@dalthviz So, I just tried it with the directory command but nothing really happens? It shows Spyder trying to load with the percentage and then nothing. I also tried running it in admin mode and it doesn't really make the folder either. It would be nice for the installer to also just show the loading bar of it being installed without the user having to mess with anything similar to how Python's installer works.

Attached a GIF of what that looks like:
![SpyderSilentInstall](https://github.com/user-attachments/assets/8676caa7-62ad-451c-8c99-ea1e0910912a)
